src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no,
+            initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img 
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        class="product-card__photo"
+      >
+
+      <div class="product-card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+
+      <div class="product-card__code">
+        Код товара: <span>195434</span>
+      </div>
+
+      <div class="product-card-rating">
+        <div class="product-card-rating-stars">
```

Comment:
```suggestion
        <div class="product-card__stars stars--4">
```
Тут еще должен быть класс модификатор, переиспользуй код из `layout_stars`